packages/plugin-core/src/_extension.ts

```diff
@@ -451,8 +451,20 @@ export async function _activate(
 
   const { keybindingConfigPath, migratedKeybindings } = KeybindingUtils.checkAndMigrateLookupKeybindingIfExists();
   if (!_.isUndefined(migratedKeybindings)) {
-    fs.ensureFileSync(keybindingConfigPath);
-    fs.writeFileSync(keybindingConfigPath, JSON.stringify(migratedKeybindings));
+    fs.copyFileSync(keybindingConfigPath, `${keybindingConfigPath}.old`);
+    writeJSONWithComments(keybindingConfigPath, migratedKeybindings);
+    vscode.window.showInformationMessage("Keybindings for lookup has been updated.", ...["Open keybindings.json"]).then((selection) => {
```

Comment:
notice sometimes vscode will not show two information boxes back to back. can we combine this into one message? `Keybindings for lookip has been updated. ["Open keybindings.json", "Open "backup"]

I think you can also adjust the timeout on the dialogue. Lets have it not fade unless the user clears it

Reply:
Looks like the message won't time out until an action is selected (a button is clicked), but can't make it wait for both buttons to be clicked.

I'll consolidate the two buttons into one so that both files are open in a horizontal split (since you want to look at both of them at the same time anyways)